### PR TITLE
[zh] Sync #14362 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14362 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md [hunk 1]
```diff
@@ -558,6 +582,8 @@ $ kubectl delete destinationrule egressgateway-for-cnn
             mode: MUTUAL
             credentialName: client-credential # 这必须与之前创建的用于保存客户端证书的 Secret 相匹配
             sni: my-nginx.mesh-external.svc.cluster.local
+            # subjectAltNames: # 如果证书是同意上一节中指定的 SAN 生成的，则可以被启用
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md:585]:
```suggestion
            # subjectAltNames: # 如果证书是随着上一节中指定的 SAN 生成的，则可以被启用
```
是不是打错字了

**Review by windsonsea — COMMENTED** [review]
Find a typo

File: content/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md]: Yes, thank you so much.

**Review by zirain — APPROVED** [review]

**Pull request merged**